### PR TITLE
Apply policy scope to permission forms displayed to project admins

### PR DESCRIPTION
**Pull request by pjanik**
[#127660867]

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -1294,11 +1297,13 @@
     t.datetime "updated_at",                      :null => false
     t.integer  "bundle_logger_id"
     t.integer  "console_logger_id"
+    t.string   "key"
```
Comment (scytacki) [inline db/schema.rb]: I don't have this "key" in my schema.rb. I suspect that your database had some migrations run on it that didn't make it into master. The new fields on interactives are legitimate, but the others changes here seem to be old. Perhaps just take this schema.rb change out of this PR. Then configure a new blank db, run the migrations and make a new PR with the missing schema bits.

**Comment by scytacki** [comment]
I went ahead and made that schema PR I described above: https://github.com/concord-consortium/rigse/pull/291

I didn't actually do this from scratch but very close, and these were the only changes missing.

**Comment by pjanik** [comment]
Thanks, I removed all schema.db changes from this PR and rebased it on top of master with your PR merged in.

**Comment by scytacki** [comment]
👍

**Pull request merged**